.changeset/nine-clouds-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+'example-backend-next': patch
```

Comment:
looks like we can delete this changeset as this package is internal

Reply:
Removed!